nextcloudappstore/core/models.py

```diff
@@ -39,6 +39,10 @@ class App(TranslatableModel):
                                             verbose_name=_('Co-Maintainers'),
                                             related_name='co_maintained_apps',
                                             blank=True)
+    recommendations = models.ManyToManyField('auth.User',
```

Comment:
Right, you're in the wrong PR though, app upload is not finished yet ;)

Can you create a new ticket?